testdata/rust/plain/repository.json

```diff
@@ -0,0 +1,13 @@
+{
+  "tasks": ["write-tests"],
+  "ignore": ["src/lib.rs"],
+  "tests-in-source": true,
```

Comment:
TODO the prompt in the test drive was:

Given the following Rust code file "src/plain.rs", provide a **test file** for this code.
The tests should produce 100 percent code coverage and must compile.
The response must contain only the test code in a fenced code block and nothing else.

```rust
pub fn plain() {
    // This does not do anything but it gives us a line to cover.
}
```